qiskit_experiments/hamiltonian_tomography/cr_hamiltonian_experiment.py

```diff
@@ -0,0 +1,299 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+Cross resonance Hamiltonian tomography.
+"""
+
+from typing import Tuple, List, Optional, Union
+
+import numpy as np
+from qiskit import QuantumCircuit, pulse, circuit
+from qiskit.exceptions import QiskitError
+from qiskit.providers import Backend
+from qiskit.providers.options import Options
+from qiskit.utils import apply_prefix
+
+from qiskit_experiments import BaseExperiment
+from .cr_hamiltonian_analysis import CRHamiltonianAnalysis
+
+
+class CRHamiltonianTomography(BaseExperiment):
+    r"""Cross resonance Hamiltonian tomography experiment.
+
+    This experiment assumes the 2 qubit Hamiltonian in the form
+
+    .. math::
+
+        H = \frac{I \otimes A}{2} + \frac{Z \otimes B}{2}
+
+    where :math:`A` and :math:`B` are Pauli operator :math:`\in {X, Y, Z}`.
+
+    This experiment is performed by stretching a pulse duration of the cross resonance
+    and measuring the target qubit by projecting onto x, y, and z basis.
+    Control qubit is prepared in both ground and excited state and the experiment is
+    repeated for both control qubit states.
+    See [SarahSheldon2016]_ for more technical details.
```

Comment:
Can you illustrate this with a single-template template circuit with pre and post rotations and then describe these in the text?